assets/js/modules/analytics-4/utils/data-mock.js

```diff
@@ -79,15 +79,15 @@ const ANALYTICS_4_DIMENSION_OPTIONS = {
 		'United States',
 		'United Kingdom',
 		'India',
-		'(not set)',
+		'Portugal',
```

Comment:
Please revert this change. We still want to mock the `(not set)` value being returned in the mock report data, as we don't filter it out in all of our reports.

```suggestion
		'(not set)',
```